src/pages/Cart/Cart.js

```diff
@@ -1,8 +1,181 @@
 import React from 'react';
 import styled from 'styled-components';
+import { RiDeleteBinLine } from 'react-icons/ri';
 
 const Cart = () => {
-  return <p>hello cart</p>;
+  return (
+    <WrapperContent>
+      <HeadContent>
+        <HeadTitle>SHOPPING CART</HeadTitle>
+        <PurchaseBtn>구매</PurchaseBtn>
+      </HeadContent>
+      <CartMainWrapper>
+        {CART_DATA.map(item => {
+          return (
+            <ItemWrapper>
+              <ItemTextWrapper>
+                <ItemTitle>{item.title}</ItemTitle>
+                <ItemText>{item.sub}</ItemText>
+              </ItemTextWrapper>
+              <ItemRight>
+                <ItemImg src="images/Cart/cat.jpg" />
+                <ItemPrice>{item.price}원</ItemPrice>
+                <DeleteBtn>
+                  <RiDeleteBinLine />
+                </DeleteBtn>
```

Comment:
기존의 `RiDeleteBtnLine`은 사용할 필요 없이 DeleteBtn을 기존의 아이콘처럼 사용하시면 됩니다.
```suggestion
                <DeleteBtn />
```

Reply:
넵 수정하였습니다